src/Routing/Route.php

```diff
@@ -203,8 +203,12 @@ protected function optionsApplyToControllerMethod(array $options)
             return true;
         } elseif (isset($options['only']) && in_array($this->method, $this->explodeOnPipes($options['only']))) {
             return true;
-        } elseif (isset($options['except']) && in_array($this->method, $this->explodeOnPipes($options['except']))) {
-            return false;
+        } elseif (isset($options['except'])) {
+            if (in_array($this->method, $this->explodeOnPipes($options['except']))) {
```

Comment:
Why not just `return ! in_array($this->method, $this->explodeOnPipes($options['except']));` ?